packages/template/package.json

```diff
@@ -7,7 +7,7 @@
   },
   "description": "> TODO: description",
   "author": "OpenTelemetry Authors",
-  "homepage": "https://github.com/open-telemetry/opentelemetry-js#readme",
+  "homepage": "https://github.com/open-telemetry/opentelemetry-js/tree/main/packages/template",
```

Comment:
This is a template package and will not be published to npm. Maybe we should add a TODO mark here to indicate that this field needs to be updated when applying the template.